### DISCUSSION TITLE
Add isTablet Runtime Variable for Improved Device Detection

### DISCUSSION THREAD

**Discussion by avega99** [jpudysz/react-native-unistyles#427] [Feature Requests]
It would be highly beneficial to introduce a runtime variable isTablet, similar to isPortrait and isLandscape. This addition would enable seamless detection of tablet devices at runtime, enhancing adaptability and improving user experience across different screen types.

**Comment by jpudysz** [comment]
Hey, shouldn't be hard to add! 
Just wondering when it's helpful if you have media queries and breakpoints 🤔 
If you target both tablet and web, then it would require some extra logic to target both.

**Reply by avega99** [reply]
Thanks for considering it! While media queries and breakpoints are great, having an isTablet runtime variable can simplify logic, especially in cases where behavior needs to be adjusted dynamically at runtime rather than just through styling.

For example, I’m targeting both tablet and web, but tablets often require unique interactions or layouts that differ from desktops, even when they share breakpoints. Having isTablet as a runtime variable would eliminate the need for additional custom logic to differentiate them

**Reply by jpudysz** [reply]
I see, make sense 🙏 . I like the idea. I will implement it somewhere in the near future.

**Reply by avega99** [reply]
Thank you so much!
